### DISCUSSION TITLE
Reasonable undervolting limits?

### DISCUSSION THREAD

**Discussion by Thrakios** [seerge/g-helper#2336] [Ideas]
Dear all,

My G14 takes the maximum amount of undervolt (-40) with perfect stability. I have tested benchmarking, and used the computer for weeks.

Though I was wondering two things: 
- Up to how much undervolting is reasonable on a laptop CPU?
- Would it be reasonable to allow for more than -40?

`
3/29/2024 10:38:54 PM: UV: -40 OK
`

All the best.

**Comment by seerge** [comment]
@Thrakios hello, 7xxx and 8xxx series AMD cpus have some kind of crash-protection from undervolting, so generally no matter how low you would set - it will keep running .  But you just won't see any additional performance gains from that after certain value.

**Reply by seerge** [reply]
And you can set any undervolting limits you want https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-cpu-undervolting-limit

**Reply by Thrakios** [reply]
That's quite amazing! I am impressed.
I guess in that case, the best approach to undervolting is benchmarking to see which setting works best.

Thanks!

**Reply by Feyce** [reply]
I woul love to know the answer about 9xxx series, since i have a g14 2021 with 5900HS
